### DISCUSSION TITLE
[VOTE] Release Apollo Java version 2.3.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo-java#76] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo Java version 2.3.0.

Release Note:

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.3.0

Maven 2 staging repository:

- https://oss.sonatype.org/service/local/repositories/comctrip-1115/content/com/ctrip/framework/apollo/

You may add the following to pom.xml to test the new artifacts.

```xml
<repositories>
    <repository>
        <id>sonatype-nexus-release</id>
        <url>https://oss.sonatype.org/service/local/repositories/comctrip-1115/content</url>
        <releases>
            <enabled>true</enabled>
        </releases>
        <snapshots>
            <enabled>true</enabled>
        </snapshots>
    </repository>
</repositories>
```

Release Tag :

 * https://github.com/apolloconfig/apollo-java/releases/tag/v2.3.0

Release CommitID :

 * https://github.com/apolloconfig/apollo-java/commit/1fc9d27940c555431eabdd8325096c8daf1812db

Guide to build the release from source:

 * mvn clean package -DskipTests=true

Voting will start now (2024/08/10) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by hezhangjian** [comment]
+1, I have checked
- build from source success
- java8 run unit test success on mac m1

**Comment by Anilople** [comment]
+1 Release this package

**Comment by zouyx** [comment]
+1 binding to release

I've checked 
- [x] All uts passed.
- [x] Release Note is fine.
- [x] Download links are fine.
- [x] Shasum are correct.

**Comment by lepdou** [comment]
+1 Release this package

**Comment by klboke** [comment]
+1, I checked
 - All uts passed.
 - Release Note is fine.

**Comment by nobodyiam** [comment]
The vote to release Apollo Java version 2.3.0 has passed with five +1 and zero -1.
